test/integration/multinode_test.go

```diff
@@ -493,9 +493,10 @@ func validateDeployAppToMultiNode(ctx context.Context, t *testing.T, profile str
 	}
 	podIPs := strings.Split(strings.Trim(rr.Stdout.String(), "'"), " ")
 	if len(podIPs) != 2 {
-		t.Errorf("expected 2 Pod IPs but got %d", len(podIPs))
+		t.Errorf("expected 2 Pod IPs but got %d, output: %q", len(podIPs), rr.Output())
 	} else if podIPs[0] == podIPs[1] {
-		t.Errorf("expected 2 different pod IPs but got %s and %s", podIPs[0], podIPs[0])
+		t.Errorf("expected 2 different pod IPs but got %s and %s. output: %q", podIPs[0], podIPs[0], rr.Output())
```

Comment:
```suggestion
		t.Errorf("expected 2 different pod IPs but got %s and %s. output: %q", podIPs[0], podIPs[1], rr.Output())
```
Outputting the same IP twice